Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -3735,8 +3734,8 @@ void CGame::Packet_VehicleTrailer(CVehicleTrailerPacket& Packet)
     CPlayer* pPlayer = Packet.GetSourcePlayer();
     if (pPlayer && pPlayer->IsJoined())
     {
-        // Spawned?
-        if (pPlayer->IsSpawned())
+        // Alive?
+        if (!pPlayer->IsDead())
```

Comment:
I see you has taken care of two call sites where IsSpawned was used. But there are more of them: 
![spawned](https://github.com/multitheftauto/mtasa-blue/assets/12121551/c095e3c4-65eb-411c-b094-219b5d546c14)

Reply:
That's important because you have changed the meaning of `m_bSpawned`.

Reply:
It's become complicated. What if you revert the last commit and just rename m_bHasPlayerSpawned with something like m_bAlive? Simple and no collisions.